pkg/dynamicinstrumentation/diconfig/config_manager.go

```diff
@@ -289,10 +293,10 @@ func newConfigProbe() *ditypes.Probe {
 		FuncName: "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer.passProbeConfiguration",
 		InstrumentationInfo: &ditypes.InstrumentationInfo{
 			InstrumentationOptions: &ditypes.InstrumentationOptions{
-				ArgumentsMaxSize:  100000,
-				StringMaxSize:     30000,
+				ArgumentsMaxSize:  50000,
+				StringMaxSize:     10000,
 				MaxFieldCount:     int(ditypes.MaxFieldCount),
-				MaxReferenceDepth: 8,
+				MaxReferenceDepth: 10,
```

Comment:
Do we need this depth for the config probe?

Reply:
No, i'm not sure why I changed it.